content/_posts/falix/dashboard/2024-10-18-external-start.md

```diff
@@ -17,30 +17,34 @@ author:
 
 The external start feature is a great way to allow players or users to start your server when needed without needing access to your server's console.
 
-## :flashlight: Toggling the External Start Feature:
-
-1. Log in to the [Dashboard](https://client.falixnodes.net/).
-
-2. Navigate to your [Profile Settings](https://client.falixnodes.net/profile/settings) by clicking the user icon in the top navigation bar then clicking on "Settings".
-
-3. Look for the "External Server Start" option under "Server Settings".
-
-4. Select "Enable" to enable the feature across all your servers or "Disable" to disable it altogether.
-
-5. Click on "**Update**{: .blue }" to apply your changes.
-
 ## :green_circle: Starting a Server Using External Start:
+{: .warning}
+> It is not allowed for server owners to use this feature, start your free server in [the console](https://client.falixnodes.net/server/console) instead.
 
-1. Go to the [External Start page](https://falixnodes.net/start).
+1. Go to the [External Start page](https://falixnodes.net/startserver).
 
 2. Enter the IP address of the server you want to start.
 
     > Specifying the port is not necessary.
 
-3. Click on the "**Start Server**{: .blue }" button to initiate a server start.
+    > you can prefill the subdomain box by adding `#name=yoursubdomain.falixsrv.me` to the end of the page url.
+
+3. Click on the "**Start Server**{: .purple }" button to initiate a server start.
```

Comment:
I currently changed this to purple beacuse the actual button is purple but IDK if it looks good...